### ISSUE TITLE
Unstable when multiple colliders spawned as rigid body children

### ISSUE THREAD

**Issue by dafteran4** [Jondolf/bevy_xpbd#244]
This is a simple example using multiple colliders spawned as children for one dynamic rigid body. It produces exploding / unstable behavior. Version is 0.3.2. It is also unstable if only one children is spawned. It is, however, fine if collider is part of the same entity as dynamic body.

```
use bevy::prelude::*;
use bevy_xpbd_3d::prelude::*;

fn main() {
    App::new()
        .add_plugins((
                DefaultPlugins,
                PhysicsPlugins::default(),
                PhysicsDebugPlugin::default(),
        ))
        .add_systems(Startup, test)
        .run();
}

fn test(mut commands: Commands) {

    let n = 10;
    let size = 20.;

    let mut heights = Vec::new();
    heights.resize_with(n, || {
        let mut v = Vec::new();
        v.resize(n, 0.);
        v
    });

    commands.spawn((
            Collider::heightfield(heights, Vec3::splat(size)),
            RigidBody::Static,
    ));

    commands
        .spawn((RigidBody::Dynamic, SleepingDisabled))
        .with_children(|c| {
            c.spawn((Collider::capsule(1.0, 0.1), Transform::from_xyz(0.0, 1.0, 0.0), ColliderDensity(100.)));
            c.spawn((Collider::capsule(1.0, 0.1), Transform::from_xyz(0.05, 2.0, 0.0), ColliderDensity(200.)));
        }); 

    commands.insert_resource(Gravity(Vec3::NEG_Y * 9.8));

    commands.spawn(Camera3dBundle {
        transform: Transform::from_xyz(0., 2., 7.).looking_at(Vec3::ZERO, Vec3::Y),
        ..default()
    });
    
}


```

**Comment by dafteran4** [comment]
Also tested this on main branch 6444910 , it behaves the same.

**Comment by mbrea-c** [comment]
The rigidbody has an offset center of mass so the problem might be related to #246. I'm working on a PR but it's running into some issues atm.

**Comment by Jondolf** [comment]
Should be fixed by #250.

**Comment by dafteran4** [comment]
Thanks. It works now.
